locales/locales.php

```diff
@@ -972,6 +1010,8 @@ public function __construct() {
 		$gsw->country_code = 'ch';
 		$gsw->wp_locale = 'gsw';
 		$gsw->slug = 'gsw';
+		$gsw->variant_root = $de->slug;
```

Comment:
Thanks @swissspidy, I'll remove the relationship.

@pixelverbieger I'm sure there will be many new variants added, like de_DE_formal once the the PR is merged, currently swisspidy is correct that the formal variants are handled by translate.w.org separately.